app/build.gradle

```diff
@@ -22,6 +22,10 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     testCompile 'junit:junit:4.12'
+    compile project(':android-animation-arsenal')
     compile 'com.android.support:appcompat-v7:23.1.0'
     compile 'com.android.support:design:23.1.0'
+    compile 'com.android.support:support-annotations:23.1.0'
+    // View Injection
+    compile 'com.jakewharton:butterknife:7.0.1'
```

Comment:
(totally don't remove this in this PR, if you decide you want to remove it, do it in another PR)